RELEASE_NOTES.rst

```diff
@@ -21,6 +21,113 @@
 
 .. towncrier release notes start
 
+Airflow 2.5.2 (2023-03-13)
+--------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+The date-time fields passed as API parameters or Params should be RFC3339-compliant (#29395)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+  In case of API calls, it was possible that "+" passed as part of the date-time fields were not URL-encoded, and
+  such date-time fields could pass validation. Such date-time parameters should now be URL-encoded (as ``%2B``).
+
+  In case of parameters, we still allow IS8601-compliant date-time (so for example it is possible that
+  ' ' was used instead of ``T`` separating date from time and no timezone was specified) but we raise
+  deprecation warning. (#29395)
+
+Default for ``[webserver] expose_hostname`` changed to ``False`` (#29547)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+  The default for ``[webserver] expose_hostname`` has been set to ``False``, instead of ``True``. This means administrators must opt-in to expose webserver hostnames to end users. (#29547)
```

Comment:
```suggestion
The default for ``[webserver] expose_hostname`` has been set to ``False``, instead of ``True``. This means administrators must opt-in to expose webserver hostnames to end users.
```